x/ibc/transfer/keeper/msg_server.go

```diff
@@ -32,13 +33,26 @@ func (k Keeper) Transfer(goCtx context.Context, msg *types.MsgTransfer) (*types.
 		return k.Keeper.Transfer(sdk.WrapSDKContext(ctx), msg)
 	}
 
+	pair, _ := k.erc20Keeper.GetTokenPair(ctx, pairID)
+
+	if !pair.Enabled {
+		// no-op: pair is not enabled so we can proceed with regular transfer
+		return k.Keeper.Transfer(sdk.WrapSDKContext(ctx), msg)
+	}
+
 	sender, err := sdk.AccAddressFromBech32(msg.Sender)
 	if err != nil {
 		// NOTE: shouldn't happen as the receiving address has already
 		// been validated on ICS20 transfer logic
 		return nil, sdkerrors.Wrap(err, "invalid sender")
 	}
 
+	senderAcc := k.accountKeeper.GetAccount(ctx, sender)
```

Comment:
nit, use the `IsModuleAccount` function instead

Reply:
`IsModuleAccount` takes accountKeeper interface-compatible object from the `erc20` module types as argument, so I can't use `ibc/transfer` accountKeeper(as interfaces differs). 
Can't figure out, maybe there are more straightforward way to determine is account a module or not?

Reply:
you can update the interface

Reply:
I moved `GetAccount()` out of helper function to not pass keeper as argument

Reply:
I can update the interface too but than we will have to make every module's `accountKeeper` interface consistent with `erc20`'s one to be compatible with this helper, so It's better to just simplify helper and make `accountsKeeper` dependant calls out of this function

Reply:
@fedekunze 
https://github.com/evmos/evmos/pull/1119/commits/049dd55ccfd0fa0ac0beae8caced7938612aff90#diff-6a8bf0e8b1256f680160460ce52ea20d49e1000d3f0a2bc0887b07cdd0ebd85dR87

What do you think on that?